rtgui/editorpanel.cc

```diff
@@ -105,7 +105,7 @@ bool find_default_monitor_profile(GdkWindow *rootwin, Glib::ustring &defprof, Gl
     // * Copyright (C) 2006 John Ellis
     // * Copyright (C) 2008 - 2016 The Geeqie Team
     // 
-    guchar *prof;
+    guchar *prof = NULL;
```

Comment:
@agriggio Please use `nullptr` instead. I converted all places with `NULL` prior to the 5.0 release and it should stay consistent from now on.